NEWS

```diff
@@ -168,6 +168,7 @@
 * Fixes in reachability of loopback ifaces
 * Fixes addressing  possible uninitialized memory accesses
 * Fixes in error flow for endpoints created upon receiving connection request
+* Fixes TCP keepalive false positive error detections
```

Comment:
Fixes in TCP keepalive to avoid false-positive error detection

Reply:
done